AlertaDengue/dados/views.py

```diff
@@ -1468,11 +1470,11 @@ def get_regional_info(
         last_year_week = None
 
         for regional_name in regional_names:
-            cities = dbdata.get_cities(
+            cities = RegionalParameters.get_cities(
```

Comment:
Here would be the most critical point...

Reply:
I didn't get much difference in caching the dictionary, it would be better to redo the maps for the states report page.
https://github.com/esloch/AlertaDengue/blob/update-regional-saude/AlertaDengue/dados/views.py#L1473